harness/tests/experiment/pytorch/test_pytorch_context.py

```diff
@@ -50,3 +53,24 @@ def test_wrap_scaler(self) -> None:
         scaler = torch.cuda.amp.GradScaler()  # type: ignore # GradScaler.__init__ is untyped
         assert scaler == self.context.wrap_scaler(scaler)
         assert scaler == self.context._scaler
+
+    def test_context_method(
```

Comment:
What meaningful test coverage are we getting here?

Also, even if we keep this test, `test_context_method` is not nearly clear enough about what this code is testing.

Reply:
I changed this test to be more useful and clear, but I'm not opposed to removing it.

Reply:
yeah I don't see that it adds much, I'd recommend removing it.